### PR TITLE
[4/4] Reserved token CSV export for v1

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

As per title.

## Screenshots or screen recordings

Screen Shot 2022-08-25 at 12 12 40 PM

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/v1/V1Project/V1ProjectToolsDrawer/ExportTicketModsButton.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      `${parseFloat(permyriadToPercent(`${mod.percent}`)) / 100}`,
+      `${mod.preferUnstaked}`,
+      `${mod.lockedUntil}`,
+    ]
+  })
+
+  return [CSV_HEADER, ...csvContent]
+}
+
+export function ExportTicketModsButton() {
```
Comment (tomquirk) [inline src/components/v1/V1Project/V1ProjectToolsDrawer/ExportTicketModsButton.tsx:31]: There is some duplication here between this and ExportPayoutModsButton. I decided it wasn't worth abstracting at this stage.

File: src/components/v1/V1Project/V1ProjectToolsDrawer/ExportTicketModsButton.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  'percent',
+  'preferUnstaked',
+  'lockedUntil',
+]
+
+const prepareTicketModsCsv = (mods: TicketMod[]): (string | undefined)[][] => {
+  const csvContent = mods.map(mod => {
+    return [
+      mod.beneficiary,
+      `${parseFloat(permyriadToPercent(`${mod.percent}`)) / 100}`,
```
Comment (tomquirk) [inline src/components/v1/V1Project/V1ProjectToolsDrawer/ExportTicketModsButton.tsx:22]: this means values are represented in fractions. i.e `0.0304` = 3.04%

**Comment by johnnyd-eth** [comment]
Should we include the remaining owner % in the export if it exists?

This...
Screen Shot 2022-08-25 at 2 54 11 pm
 ...results in...
Screen Shot 2022-08-25 at 2 54 36 pm

**Review by johnnyd-eth — APPROVED** [review]
LGTM

**Pull request merged**